docs/content/functioneel/zaken_api.md

```diff
@@ -0,0 +1,104 @@
+| Consumer        | Componenten waarmee zaken behandeld worden (Zaakafhandelcomponenten; ZAC’s) en (andere) componenten die zaakgegevens raadplegen.                                                                  |
+| Informatiemodel | [RGBZ, versie 2.00.02](https://www.gemmaonline.nl/images/gemmaonline/a/a4/EgIM_20180621_-_ag_4_-_GEMMA_RGBZ_2.0_-_CONCEPT_20180613.zip) (in-ontwikkeling)                                         |
+| Specificaties   | <https://ref.tst.vng.cloud/zrc/api/v1/schema>                                                                                                                                                     |
+| Bijzonderheden  | Het RGBZ wordt in meerdere API’s uitgewerkt. De versie van het RGBZ waarnaar verwezen wordt omvat meer dan hetgeen in deze API is uitgewerkt.                                                     |
+
+Deze API omvat (vooralsnog) de resources Zaken, Rollen en Statussen.
+
+Elke resource biedt de mogelijkheid om objecten toe te voegen (POST), te wijzigen (PUT, PATCH), te verwijderen (DELETE) en te raadplegen (GET), tenzij anders vermeld.
+
+### Resource: Zaken
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                                                                                                                                                                       |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een zaak toevoegen, gegevens van een zaak wijzigen, wijzigingen aanbrengen in de relatie van een zaak tot betrokkenen en/of objecten dan wel één of meer gegevens van zaken en genoemde relaties opvragen.                                                                                                            |
+| Voorwaarde     | 1. Het zaaktype van de zaak moet aanwezig en geldig zijn (resource: zaaktypen in API: ZTC). <br>2.  Een te relateren Betrokkene moet aanwezig zijn (resource: [...] in API:     [...]). <br>3.  Een te relateren Object moet aanwezig zijn (resource: [...] in API: [...]). <br>4.  Bij het toevoegen van een zaak mag de (functionele) Identificatie meegegeven worden; deze dient uniek te zijn. |
+| Gevolg         | De zaak wordt, indien sprake is van valide gegevens, toegevoegd aan (bij een POST) dan wel gewijzigd in (PUT, PATCH) de Zaakregistratiecomponent dan wel worden (bij GET) gegevens van de gevonden zaken beschikbaar gesteld. Indien bij een POST geen functionele identificaties zijn meegeleverd dan wordt deze door de ZRC bepaald. |
```

Comment:
De zaak wordt toegevoegd aan .. wat? Wordt de zaak niet gewoon aangemaakt in het ZRC?
Misschien beter: Indien bij het aanmaken van een zaak geen zaakidentificatie wordt meegeleverd wordt deze door de ZRC bepaald.

Reply:
Toegevoegd aan ... de Zaakregistratiecomponent. Dat staat er nu. Ik zou zoveel mogelijk dezelfde termen willen gebruiken: toevoegen, wijzigen, etcetera. Of is toevoegen niet de juiste term? 
Die laatste regel pas ik aan, al weet ik nog niet hoe. 'aanmaken' of 'toevoegen'?

Reply:
Er is zekers iets, een ZRC met nul of meer zaken. Ik heb moeite met het woord 'aanmaken'. Ik heb er nu 'toegevoegd door de ZRC' van gemaakt. Dus niet meer 'in'.